cli/src/server.js

```diff
@@ -164,7 +164,8 @@ program
     '-d, --down <down>',
     "Optional, runs down migration down to [down file name], also accepts ['last']"
   )
-  .option('-i, --info [info]', "Display the state of the migrations, optional ['v'|'verbose']");
+  .option('-i, --info [info]', "Display the state of the migrations, optional ['v'|'verbose']")
+  .option('-f, --fixOrder [fixOrder]', 'Resolves the migration order conflicts');
```

Comment:
Did you reproduce the original issue below?

> Error: Db migration (20180212_expiration) does not match (20180320093000_shareable_dashboards)

If yes, did you verify that this fixes that issue?